ui/app/templates/components/masked-input.hbs

```diff
@@ -18,7 +18,7 @@
   {{#if allowCopy}}
     <CopyButton
       @clipboardText={{value}}
-      @class="copy-button button is-compact"
+      @class="copy-button button {{if displayOnly "is-compact"}}"
```

Comment:
this should fix the alignment issue mentioned above. i used the pattern i saw with the toggle button.

i tried initially just re-adding the `is-compact` class, but doing so set the height and padding of the button which caused it to be misaligned with the password input and toggle button. i also tried overwriting these styles by adding `height: unset` and `padding: unset` (also tried `auto`) to  the `copy-button` class, but that didn't work either.

i checked that the alignment still looks okay on the secrets show page so hopefully this solution works!